tailwind.config.js

```diff
@@ -10,27 +10,16 @@ module.exports = {
   ],
   theme: {
     colors: {
-      primary: '#ce283d',
-      'dark-red': '#a52a2a',
-      transparent: 'transparent',
-      current: 'currentColor',
+      primary: {
+        DEFAULT: '#ce283d',
+        dark: '#a52a2a'
+      },
+      transparent: colors.transparent,
+      current: colors.current,
+      white: colors.white,
+      black: colors.black,
       gray: colors.gray,
-      cream: '#f9f9f9',
-      white: '#ffffff',
-      black: '#000000',
-      facebook: '#4267B2',
-      twitter: '#00ACEE',
-      googleblue: '#4285F4',
-      googlegreen: '#34A853',
-      googleyellow: '#FBBC05',
-      googlered: '#EA4335',
-      applegrey: '#A2AAAD',
-      microsoftblue: '#00A4EF',
-      yahoopurple: '#430297',
-      'dark-gray': '#7B7B7B',
-      'zinc-200': '#F2F2F2',
-      'zinc-300': '#D9D9D9',
-      'zinc-50': '#FFFFFF'
+      zinc: colors.zinc
```

Comment:
Do we need both 'gray' and 'zinc'? https://tailwindcss.com/docs/customizing-colors#default-color-palette